docs/man/docker-create.1.md

```diff
@@ -60,9 +61,12 @@ IMAGE [COMMAND] [ARG...]
 **--cidfile**=""
    Write the container ID to the file
 
-**--cpuset**=""
+**--cpuset-cpus**=""
    CPUs in which to allow execution (0-3, 0,1)
 
+**--cpuset-mems**=""
+   MEMs in which to allow execution (0-3, 0,1)
```

Comment:
mmm, i think you will need to tell the reader what an `MEMs` is - CPU possibly not - @fredlf ?